altymis-AVSCP

altymis opened the pull request.

kathsherratt commented:
Hi @altymis - it looks like this formatting is correct, so I think the problem is the csv has a blank last line which is causing the checks to fail.
Kath

altymis commented:
Hi,

blank lines in .csv - fatal error, I will fix and try again.
As for the test verification - all the files that are used for this test -
are on my local computer.
Of course - I can send you a few hundred files for manual verification of
the dates they were saved - but that's a little crazy, don't you think? :-)
:-)


MK

sob., 10 kwi 2021 o 17:10 Kath Sherratt ***@***.***>
napisał(a):

> Hi @altymis <https://github.com/altymis> - it looks like this formatting
> is correct, so I think the problem is the csv has a blank last line which
> is causing the checks to fail.
> Kath
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/epiforecasts/covid19-forecast-hub-europe/pull/227#issuecomment-817151316>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATPHVLTFBQS7ZQ4WWY4S5Z3TIBS75ANCNFSM42WSQ66Q>
> .
>

altymis commented:
Oh ... I don't understand ... I corrected the csv file, deleted the empty
line, I follow the exact procedure - and it doesn't work, again the same
error with pull request...

sob., 10 kwi 2021 o 18:56 github-actions[bot] ***@***.***>
napisał(a):

> Your submission has some validation errors. Please check the logs of the
> build under the "Checks" tab to get more details about the error.
>
> Your submission has some metadata validation errors. Please check the logs
> of the build under the "Checks" tab to get more details about the error.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/epiforecasts/covid19-forecast-hub-europe/pull/227#issuecomment-817170573>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATPHVLRKULRHMBXPUBFCHS3TIB7LRANCNFSM42WSQ66Q>
> .
>

team_name: altymis
model_name: AVSCP
model_abbr: altymis-AVSCP
model_contributors: Mariusz Kwiatkowski, ***@***.***>
website_url: https://docs.google.com/spreadsheets/d/1TfTuLVZNphfEbFgSvAtC_zprdE6ifsr9YTJNjDPTrRY/edit?usp=sharing
license: cc-by-nc-4.0
team_model_designation: proposed
methods: Converting the relationship between two groups of phrases in Google Trends related to COVID.
team_funding: private project, not commercial
twitter_handles: altymis
data_inputs: Google Trends data

methods_long: "A method of calibrating the relationship between groups\
\ of phrases (values for phrases taken from Google Trends) regarding symptoms\
\ and searching for treatment and testing methods.\
\ The starting point was the number of cases in Poland on 2020-10-12.\ 
\ All subsequent values result only from conversions concerning\
\ the weight of the relationship between groups of phrases.\
\ In order to properly determine the burden on the health care\
\ system, a shift by 14-21 days should be taken into account -\ 
\ the numbers of cases "shift over time" due to various personal\
\ unpredictable parameters (treatment delay, problem in access to\
\ a doctor, fear of illness, etc.)\
\"
